products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,233 @@
+as well.
+
+```opencypher
+MATCH (d:Death)-[:HAPPENED_IN]->(e:Episode)
+RETURN e.name AS episode_name, COUNT(d) AS kill_count
+ORDER BY kill_count DESC
+```
+
+3) List the number of kills per season. If you have watched the show, you should guess this one.
+
+```opencypher
+MATCH (d:Death)-[:HAPPENED_IN]->(s:Season) 
+RETURN s.number AS season_number, COUNT(d) AS death_count
+ORDER BY season_number ASC
+```
+
+4) Worst season by far, based on user ratings, is the last one, but can you guess the best one?
+Let's list the seasons by IMDB ratings. The problem we get with using the `AVG()` function is that it
+gives us too many decimals, therefore a useful solution is given in this example using `ROUND()`.
+
+```opencypher
+MATCH (e:Episode)-[:PART_OF]->(s:Season) 
+RETURN s.number AS season_name, ROUND(100 * AVG(e.imdb_rating))/100 AS rating
+ORDER BY rating DESC
+```
+
+5) There are many methods by which characters were killed such as sword or Dragonfire, but let's list victims
+of unique methods.
+
+```opencypher
+MATCH (:Character)-[k:KILLED]->(:Character)
+WITH k.method AS kill_method, COUNT(k.method) AS method_count
+WHERE method_count < 2
+MATCH (killer:Character)-[k:KILLED]->(victim:Character)
+WHERE k.method = kill_method
+RETURN kill_method, victim.name AS victim
+```
+
+6) Daenerys Stormborn of House Targaryen, the First of Her Name, Queen of the Andals and the First Men,
+Protector of the Seven Kingdoms, the Mother of Dragons, the Khaleesi of the Great Grass Sea, the Unburnt,
+the Breaker of Chains or shortened to "Daenerys Targaryen" in our database is the biggest killer on
+the show. Let's list all the episodes she killed in as well as characters she killed.
+
+```opencypher
+MATCH (daenerys:Character {name: 'Daenerys Targaryen'})-[:KILLED]->(victim:Character)
+MATCH (daenerys)-[:KILLED_IN]->(d:Death)<-[:DIED_IN]-(victim)
+MATCH (d)-[:HAPPENED_IN]-(e:Episode)
+RETURN DISTINCT(victim.name) AS victim, COUNT(d) AS kill_count, e.name AS episode_name
+ORDER BY kill_count DESC
+```
+
+7) Remember that shocking last episode of the fifth season when they killed Jon Snow and we totally thought
+he was gonna stay dead? Well, let's list all the characters that would survive if he actually stayed dead.
+
+```opencypher
+MATCH (jon:Character {name: 'Jon Snow'})-[:KILLED]->(victim:Character)
+MATCH (jon)-[:DIED_IN]->(jon_death:Death)
+MATCH (jon)-[:KILLED_IN]->(victim_death:Death)<-[:DIED_IN]-(victim)
+WHERE victim_death.order>jon_death.order
+RETURN DISTINCT(victim.name) AS victim, COUNT(victim_death) AS kill_count
+ORDER BY kill_count DESC
+```
+
+8) Houses or allegiances are one of the main aspects of Westeros. Some houses killed more characters
+than others, but that doesn't matter in the end, what matters is efficiency. Let's list the allegiances with
+the best Kill/Death Ratios, or short KDR. Here we came across one additional problem, if an allegiance had more 

```

Comment:
`or short KDR` => `or KDR for short`

Also in `... problem, if an ...`, you can replace the comma with a full-stop and start a new sentence.